README.md

```diff
@@ -1,2 +1,46 @@
 # beer-is-poetic
 Beer is Poetic
+
+## Description
+
+This website allows you to search for a brewery when you enter a city name and will generate a satellite image of the city as well as a random poem that has the word "beer" in it and credits the author.
+
+3 APIs were called:
+    List of breweries by city
+    Satellite image of city
+    Random poems
+
+Built on: 
+    HTML 
+    CSS 
+    JS
+Tailwinds for bootstrap
+jQuery and jQueryUI
+
+
+## Installation
+
+N/A
+
+
+## Credits
+
+@kevinchogan, @TreyLathe, @waltermatulich, @jonsantosdev
+
```

Comment:
I don't know if we need this, but in mine I also put resources used (like naming Drew and Kyle, different documentations, Xpert, etc.